pkg/apiserver/statement/statement.go

```diff
@@ -337,22 +339,41 @@ func (s *Service) downloadTokenHandler(c *gin.Context) {
 // @Failure 401 {object} utils.APIError "Unauthorized failure"
 func (s *Service) downloadHandler(c *gin.Context) {
 	token := c.Query("token")
-	fielPath, err := utils.ParseJWTString("statements/download", token)
+	tokenPlain, err := utils.ParseJWTString("statements/download", token)
+	if err != nil {
+		utils.MakeInvalidRequestErrorFromError(c, err)
+		return
+	}
+	arr := strings.Fields(tokenPlain)
+	if len(arr) != 2 {
+		utils.MakeInvalidRequestErrorFromError(c, errors.New("invalid token"))
+		return
+	}
+	secretKey, err := base64.StdEncoding.DecodeString(arr[0])
 	if err != nil {
 		utils.MakeInvalidRequestErrorFromError(c, err)
 		return
 	}
 
-	_, err = os.Stat(fielPath)
+	filePath := arr[1]
+	fileInfo, err := os.Stat(filePath)
+	if err != nil {
+		_ = c.Error(err)
+		return
+	}
+	f, err := os.Open(filePath)
 	if err != nil {
 		_ = c.Error(err)
 		return
 	}
 
 	c.Writer.Header().Set("Content-type", "application/octet-stream")
```

Comment:
since transmit `.csv` directly now, `Content-Type` may can be `text/csv`

Reply:
I just searched there isn't a `text/csv` kind Content-Type, and `application/octet-stream` is ok for any kind of content that is used to download.

Reply:
Found it here: https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types will change it.